<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FatCRM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionImportContacts"/>
    <addaction name="separator"/>
    <addaction name="actionSynchronize"/>
    <addaction name="actionFullReload"/>
    <addaction name="actionOfflineMode"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Setti&amp;ngs</string>
    </property>
    <addaction name="actionCRMAccounts"/>
    <addaction name="separator"/>
    <addaction name="actionConfigureFatCRM"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutFatCRM"/>
    <addaction name="actionAboutKDAB"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCRMAccounts">
   <property name="icon">
    <iconset theme="network-workgroup">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Manage &amp;CRM Accounts...</string>
   </property>
   <property name="toolTip">
    <string>Manage CRM Accounts</string>
   </property>
  </action>
  <action name="actionSynchronize">
   <property name="text">
    <string>&amp;Synchronize</string>
   </property>
   <property name="toolTip">
    <string>Synchronize current CRM Account</string>
   </property>
   <property name="statusTip">
    <string>Synchronize current CRM Account</string>
   </property>
  </action>
  <action name="actionOfflineMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="network-offline">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Offline Mode</string>
   </property>
   <property name="statusTip">
    <string>Make resource offline. This stops any synchronization with the server. Local changes will be applied when going back online.</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImportContacts">
   <property name="icon">
    <iconset theme="document-import">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Contacts...</string>
   </property>
  </action>
  <action name="actionConfigureFatCRM">
   <property name="icon">
    <iconset theme="configure">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure FatCRM...</string>
   </property>
  </action>
  <action name="actionAboutFatCRM">
   <property name="icon">
    <iconset theme="fatcrm">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About &amp;FatCRM...</string>
   </property>
  </action>
  <action name="actionFullReload">
   <property name="text">
    <string>&amp;Full Reload</string>
   </property>
   <property name="toolTip">
    <string>Full Reload</string>
   </property>
   <property name="statusTip">
    <string>Reload everything from the server. This is useful after merging accounts, which does not correctly update the opportunities and accounts that were using the old account.</string>
   </property>
  </action>
  <action name="actionAboutKDAB">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/kdablogo.png</normaloff>:/images/kdablogo.png</iconset>
   </property>
   <property name="text">
    <string>About KDAB</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
